backend/src/main/java/com/google/sps/notifiers/EmailNotifier.java

```diff
@@ -0,0 +1,49 @@
+// Class used to send users emails regarding their status in regards to their Ad-Lib Meeting.
+
+import com.sendgrid.*;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.*;
+
+import java.io.IOException;
+
+
+
+public class EmailNotifier extends Notifier {
+
+    // Member Variables 
+    
+    public String toEmail ;       
+    public String fromEmail;
+
+    public EmailNotifier(String Name, String fromEmail, String toEmail)
+    {
+
+        super(Name);
+        this.toEmail = toEmail;
+        this.fromEmail = fromEmail;
+    }
+    
+    public static void main(Strings[], args) throws IOException {
+        Email from = new Email(fromEmail);
+        String subject = "Adlib Meeting Notification";
```

Comment:
nit: Ad-lib.
Subject should be something along the lines of "Ad-lib: Match found!" (since the content implies a match has been found)

Reply:
+1 to Janet's suggestion on the email subject. Also, make sure the subject is consistent across different situations, and make sure they are documented in the design doc.

Reply:
Ack